packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -143,7 +149,8 @@ export class DeltaStorageService implements IDeltaStorageService {
 		tenantId: string,
 		id: string,
 		from: number, // inclusive
-		to: number, // exclusive
+		to: number, // exclusive,
+		fetchReason: string,
```

Comment:
I believe we would need to make this parameter optional at first. I am assuming there will be some old versions of code that is not aware of the fetchReason parameter and may not provide it, so this param should be optional? @jatgarg  to confirm.

Also, the `IDeltaStorageService` class has the `fetchReason` param for get defined as optional. Should follow the interface definition.

Reply:
At first it was left as non-optional because the string at the wrapper couldn't be undefined. I added the logic so that a string is always sent but if fetchReason is empty, then the reason says "No reason provided".